README.md

```diff
@@ -280,26 +280,26 @@ sorted-order storage algorithm selection. Btree stores are
 read-optimized (Oracle, SQLServer, Postgres, SQLite2, MySQL, MongoDB,
 CouchDB), hybrid stores are read-optimized with better
 write-throughput (Tokutek MySQL/MongoDB), while LSM-variants are
-write-optimized (HBase, Cassandra, SQLite3/LSM, Cockroach).
+write-optimized (HBase, Cassandra, SQLite3/LSM, CockroachDB).
 
 ![Read vs. Write Optimization Spectrum](/resource/doc/read-vs-write.png?raw=true)
 
 ## Architecture
 
-Cockroach implements a layered architecture, with various
+CockroachDB implements a layered architecture, with various
 subdirectories implementing layers as appropriate. The highest level of
-abstraction is the SQL layer (currently not implemented). It depends
-directly on the [structured data API][5] ([structured/][6]). The structured
+abstraction is the [SQL layer][5], which is currently in development and depends
```

Comment:
you need not mention that it is in development. The entire system is in development